core/Mail/Transport.php

```diff
@@ -0,0 +1,141 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Mail;
+
+use PHPMailer\PHPMailer\PHPMailer;
+use PHPMailer\PHPMailer\SMTP;
+use Piwik\Config;
+use Piwik\Container\StaticContainer;
+use Piwik\Mail;
+use Piwik\Piwik;
+use Piwik\Version;
+
+class Transport
+{
+    /**
+     * Sends the given mail
+     *
+     * @param Mail $mail
+     * @return bool
+     * @throws \DI\NotFoundException
+     * @throws \PHPMailer\PHPMailer\Exception
+     */
+    public function send(Mail $mail)
+    {
+        $phpMailer = new PHPMailer(true);
+
+        PHPMailer::$validator = 'pcre8';
+        $phpMailer->CharSet = PHPMailer::CHARSET_UTF8;
+        $phpMailer->Encoding = PHPMailer::ENCODING_QUOTED_PRINTABLE;
+        $phpMailer->XMailer = 'Matomo ' . Version::VERSION;
```

Comment:
Any chance we find a way to white label this term if someone configured a different brand name in WhiteLabel plugin? Not too important though. Or could maybe replace with `SettingsPiwik.getMatomoUrl()`? Not sure if/what restrictions there are on this

Reply:
Suppose we'd also not want to directly disclose the version number like this maybe? Not too much of an issue though maybe.

Reply:
You can set whatever you want as XMailer header. If nothing is set it will default to `PHPMailer X.Y.Z (https://github.com/PHPMailer/PHPMailer)`. I'll change that to an whitespace string. According to the PHPMailer code, that should remove the xmailer header